include/ua_plugin_nodestore.h

```diff
@@ -233,6 +239,27 @@ typedef struct {
     UA_MethodCallback method;
 } UA_MethodNode;
 
+
+/** Attributes for nodes which are capable of generating events */
+#ifdef UA_ENABLE_SUBSCRIPTIONS_EVENTS
+/* linked list of monitoredItems */
+    typedef struct UA_MonitoredItemQueueEntry {
+        struct {
+            struct UA_MonitoredItemQueueEntry *sle_next;	/* next element */
+        } next;
+        struct UA_MonitoredItem *mon;
```

Comment:
Usually, the `next` pointer would be contained in the UA_MonitoredItem, which is opaque here.
This entire thing can be simplified by putting

```c
#define UA_EVENT_ATTRIBUTES                                         \
     UA_MonitoredItem *monitoredItemQueue;
#endif
```

All the rest can be covered inside the implementation. To keep the public API light.